src/common/io.py

```diff
@@ -1230,6 +1237,7 @@ def __init__(self, fname, delayed_trajectory_loading = True):
             self._fname = fname
             self._is_stream = True
             delayed_trajectory_loading = False
+        self._allow_file_updates = allow_file_updates
```

Comment:
Would be good to verify that allow_file_updates is a boolean here otherwise throw an exception